doc/Language/variables.pod6

```diff
@@ -222,7 +221,7 @@ old variable as we did with C<$*dynamic2>.
 The dynamic variables differ from other variable types in that referring
 to an undeclared dynamic variable is not a compile time error but a
 runtime L<failure|/type/Failure>, so a dynamic variable can be used
-undeclared as long as it is checked for definedness or used in a
```

Comment:
This sentence is way too long. I don't like the contraction either. Bonus points for splitting it into two sentences.